ansible_base/views/resource.py

```diff
@@ -0,0 +1,145 @@
+from django.db import transaction
+from django.http import HttpResponseNotFound
+from django.shortcuts import redirect
+from django_filters import rest_framework as filters
+from rest_framework import permissions, status
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework.viewsets import GenericViewSet, mixins
+
+from ansible_base.models import PostgresTransaction, Resource, ResourceType, get_registry, service_id
+from ansible_base.serializers import DestroyResourceSerializer, ResourceSerializer, ResourceTypeSerializer, TransactionSerializer, get_resource_detail_view
+from ansible_base.utils.transactions import commit_transaction, create_transaction, rollback_transaction
+
+
+class ResourceFilter(filters.FilterSet):
+    service_id = filters.CharFilter(field_name="_computed_service_id")
+    ansible_id = filters.CharFilter(field_name="_ansible_id")
+    resource_type = filters.CharFilter(field_name="content_type__resource_type__resource_type")
+
+    class Meta:
+        model = Resource
+        fields = ["name", "object_id"]
+
+
+class TransactionViewSet(
+    mixins.RetrieveModelMixin,
+    mixins.ListModelMixin,
```

Comment:
I'm not necessarily convinced that a list view of transactions is needed. I just don't want this to become harder than it needs to.